### ISSUE TITLE
default search field should search in all station attributes

### ISSUE THREAD

**Issue by xylo** [segler-alex/radiobrowser#7]
If I search for "soundtrack" I get only stations including the word in the name but not stations that are tagged with "soundtrack".  It would be nice if the default search would search in the name as well as in all attributes.

**Comment by kousu** [comment]
Yes yes yes yes yes.

I just came here to add this exact bug. If you search for [London](http://www.radio-browser.info/gui/#!/byname/london) you get some stations, but [UK](http://www.radio-browser.info/gui/#!/byname/UK) is a totally different set because the search is only looking in the name field.

This oversight in the UI is probably fueling some of the tagging confusion: https://github.com/segler-alex/radiobrowser-api/issues/56

**Comment by kyjus25** [comment]
Good idea, I'll add this feature to my redesign UI. It would be better if the backend did it, but I could totally make separate calls to each of the endpoints, shove them all into an array, and sort by popularity.
